src/test/resources/integration/select/select-config.xml

```diff
@@ -46,6 +46,15 @@
         </db:select>
     </flow>
 
+    <flow name="emptyResult">
+        <db:select config-ref="dbConfig">
```

Comment:
This test is highly dependant on nobody else adding records into SPACESHIP table for some other purpose and inadvertently making this test useless.

This should start by truncating the table.